stylelint.config.js

```diff
@@ -0,0 +1,106 @@
+"use strict"
+
+module.exports = {
+  "extends": "stylelint-config-recommended",
+  "rules": {
+    "at-rule-empty-line-before": [ "always", {
+      except: [
+        "blockless-after-same-name-blockless",
+        "first-nested",
+      ],
+      ignore: ["after-comment"],
+    } ],
+    "at-rule-name-case": "lower",
+    "at-rule-name-space-after": "always-single-line",
+    "at-rule-semicolon-newline-after": "always",
+    "block-closing-brace-empty-line-before": "never",
+    "block-closing-brace-newline-after": "always",
+    "block-closing-brace-newline-before": "always-multi-line",
+    "block-opening-brace-newline-after": "always-multi-line",
+    "block-opening-brace-space-after": "always-single-line",
+    //"block-opening-brace-space-before": "always",
```

Comment:
Please delete commented code, if this is affecting your development, just delete other than commenting it out